asn1/LedgerLedgerLotocol.asn

```diff
@@ -107,18 +107,27 @@ CALL ::= CLASS {
 CallSet CALL ::= {
     -- All requests MUST use odd typeIds
     -- All responses MUST use even typeIds
-    {1 PacketRequest} |
-    {2 PacketResponse} |
-    {3 PrepareRequest} |
-    {4 PrepareResponse} |
-    {5 FulfillRequest} |
-    {6 FulfillResponse} |
-    {7 RejectRequest} |
-    {8 RejectResponse} |
-    {9 TransferStateRequest} |
-    {10 TransferStateResponse} |
-    {11 CustomRequest} |
-    {12 CustomResponse}
+    -- {1 reserved because this is the typeId of InterledgerProtocolPaymentPacket in
+    -- https://github.com/interledger/rfcs/edit/mj-lll-packet-types/asn1/GenericPacket.asn } |
+    {2 QuoteLiquidityRequest} |
+    {3 QuoteLiquidityResponse} |
+    {4 QuoteBySourceAmountRequest} |
+    {5 QuoteBySourceAmountResponse} |
+    {6 QuoteByDestinationAmountRequest} |
+    {7 QuoteByDestinationAmountResponse} |
+    {8 InterledgerProtocolError} |
+    -- skipping numbering to allow more future ILP-layer packet types
+    {254 CustomRequest} |
+    {255 CustomResponse} |
+    -- rest of types are outside the ILP-layer, so including the ledger layer
+    {256 PrepareRequest} |
```

Comment:
My personal preference would be to number these 9-16, but since they're ledger layer and not Interledger layer, they have to go with higher numbers.
It might actually be necessary to wrap them all in "escape code" type 255, so that you get bytes:

255 ILP-packet type escape code
1      high byte of ledger-level packet type
1      low byte of ledger-level packet type
...    rest of the data, determined by ledger-level packet type.